pymc_marketing/mmm/transformers.py

```diff
@@ -260,6 +266,103 @@ def delayed_adstock(
     return batched_convolution(x, w, axis=axis)
 
 
+def weibull_adstock(
+    x,
+    lam=1,
+    k=1,
+    l_max: int = 12,
+    axis: int = 0,
+    type: WeibullType = WeibullType.PDF,
+):
+    R"""Weibull Adstocking Transformation.
+
+    This transformation is similar to geometric adstock transformation but has more degrees of freedom, adding more flexibility.
+    .. plot::
+        :context: close-figs
```

Comment:
Think there should be a space between :context: and code or it will show up in the documentation

Reply:
Also a space before `.. plot::` or that will also not render